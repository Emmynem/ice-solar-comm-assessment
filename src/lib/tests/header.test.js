import React from "react";
import renderer from "react-test-renderer";
import Header from "../header";

describe("Header", () => {
    it("renders properly", () => {
        const tree = renderer
            .create(<Header header="Login" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});