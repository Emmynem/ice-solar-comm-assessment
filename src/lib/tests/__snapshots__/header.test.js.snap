// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders properly 1`] = `
<div>
  <h1>
    Login
  </h1>
</div>
`;
